pkg/build/api/types.go

```diff
@@ -512,7 +512,7 @@ type SourceBuildStrategy struct {
 	Scripts string
 
 	// Incremental flag forces the Source build to do incremental builds if true.
-	Incremental bool
+	Incremental *bool
```

Comment:
yeah i'm not worried about the "i didn't specify it and now it's true" case... that's the decision the cluster admin is making when they enable the setting.  but the fact that i might have explicitly set it to false and that is going to get ignored is bad.